autoreject/autoreject.py

```diff
@@ -358,6 +355,8 @@ def compute_thresholds(epochs, method='bayesian_optimization',
     augment : boolean
         Whether to augment the data or not. By default it is True, but
         set it to False, if the channel locations are not available.
+    dots : tuple of ndarray
+        The self dots and cross dots
```

Comment:
this is really not a clear docstring. Can you do better? for example say that it's useful for MEG interpolation. I would also provide the shapes of the self and cross dots, say what they correspond to and say that the tuple if of length 2

Reply:
also if the dots are plain arrays is there a risk if mismatch with channel names?

mne says it's shape (n_coils, n_coils)

Reply:
Sorry about taking so long with this. We need to do another iteration on this. Because you can do something similar with EEG ... and it's our main user base